priv/migration_events/polyn.stream.create.v1/polyn.stream.create.v1.schema.v1.json

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
I tweaked this a little from the original to allow some fields to be `null` which JetStream allows. I'm thinking we should put copies of this in the Polyn-protocol repo